### ISSUE TITLE
Error trying to install anaconda libraries

### ISSUE THREAD

**Issue by newb7147** [Microsoft/PTVS#2082]
I'm trying visual studio and am getting the following error trying to install some libraries.  I was able to install them fine on my surface last night but have been having this issue on my desktop.  I've tried uninstalling and reinstalling but keep getting the same error.

[Window Title]
Python Tools for Visual Studio

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[^] Hide details  [Close]

[Expanded Information]
```
System.InvalidOperationException: Operation cannot be started because the environment is not configured.
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<AbortIfNotReady>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<InstallAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtensionProvider.<InstallPackage>d__36.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtension.<InstallPackage_Executed>d__14.MoveNext()
```

**Comment by zooba** [comment]
There's a lot of noise, but the error is saying that your Python environment isn't configured properly.

Take a look at the [Python Environments](https://github.com/Microsoft/PTVS/wiki/Python-Environments) window and see if paths look incorrect or greyed out. Possibly you'll need to select a different environment to install into, or perhaps reinstall Python/Anaconda.

**Comment by zooba** [comment]
Dup of #2495, fixed in #2502
